README.md

```diff
@@ -0,0 +1,13 @@
+# Heading level 1
+
+1. First item
```

Comment:
It looks like something went wrong, let's discuss this point and learn how to confidently do basic things with git. Feel free if you need to contact me and discuss any questions that arise. Now I see that you have coped with your first commit and pull request! This is very, very impressive, congratulations!

Now let's try to change it and do as described in the task, this will bring us additional experience and give us confidence how to make commits, push them and work with pull requests

<img width="400" alt="image" src="https://github.com/user-attachments/assets/74598390-cfb5-4dde-bc93-7c67ca498e60">

Reply:
Hello, I'll add describe.

Reply:
Great job! 

So far, we have time, how about deleting the old text, where is item 1 and some kind of table? It doesn't seem to be needed

We can also add some magic and use [markup syntax](https://docs.github.com/ru/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) for styling. I can suggest highlighting the role in bold (mentor and student), and making our names links to the account. What do you think about this?

For example:

**Student**: [Yevheniia Datsenko](https://github.com/Datcenko)



If you have a desire, you can come up with and add something of your own additionally, it will be very cool. I will also explain why I pay attention to this, when we write documentation, we use markup for this. Similarly, much of this markup works in slack or telegram. As an example, if we want to insert and highlight something, then we can use back quotes **`**: `word inner back qoutes`. If we have multilines, we will three back qoutes:
```
Before this text stay three back quotes
           save formating, a lot of spaces        before
After this text stay three back quotes

```

<img width="335" alt="image" src="https://github.com/user-attachments/assets/0da4c64c-b430-4b65-81e5-c570ea31c140">